addons/web/static/tests/legacy/mockserver_tests.js

```diff
@@ -181,7 +181,31 @@ QUnit.module("MockServer", {
             args: [[undefined, 1]],
             kwargs: {},
         });
-        assert.deepEqual(result, [[null, "False"], [1, "Jean-Michel"]]);
+        assert.deepEqual(result, [[null, ""], [1, "Jean-Michel"]]);
+    });
+
+    QUnit.test("performRpc: name_get with single id 0", async function (assert) {
+        assert.expect(1);
+        const server = new MockServer(this.data, {});
+        const result = await server.performRpc("", {
+            model: "res.partner",
+            method: "name_get",
+            args: [0],
+            kwargs: {},
+        });
+        assert.deepEqual(result, []);
+    });
+
+    QUnit.test("performRpc: name_get with array of id 0", async function (assert) {
+        assert.expect(1);
+        const server = new MockServer(this.data, {});
+        const result = await server.performRpc("", {
+            model: "res.partner",
+            method: "name_get",
+            args: [[0]],
+            kwargs: {},
+        });
+        assert.deepEqual(result, [[null, ""]]);
```

Comment:
weird, isn't it?

Reply:
It is, afeter looking into it a bit further, we should probably return [id, ""] instead of [null, ""]. We'll make another commit to address that specific issue

Reply:
On second thoughts, we should have a different behaviour for false/0 and for null/undefined, don't really know if it's worth the trouble... If so, this can be done in another PR